modules/core/test/test_arithm.cpp

```diff
@@ -1923,7 +1923,7 @@ TEST(Core_ArithmMask, uninitialized)
 
         Mat a(dims, sizes, type), a1;
         Mat b(dims, sizes, type), b1;
-        Mat mask(dims, sizes, CV_8U);
+        Mat mask(dims, sizes, CV_Bool);
```

Comment:
We should not modify existed tests. We should add new tests for new types support.

Reply:
I would make it random, 8U or bool

Reply:
Added couple of more tests.